command/commands.go

```diff
@@ -71,6 +72,8 @@ import (
 	sr "github.com/hashicorp/vault/serviceregistration"
 	csr "github.com/hashicorp/vault/serviceregistration/consul"
 	ksr "github.com/hashicorp/vault/serviceregistration/kubernetes"
+
+	hcpengine "github.com/hashicorp/hcp-vault-engine-poc"
```

Comment:
In general our style is to group all the non-Go-standard-lib into a single second block of imports.